packages/amplify-cli/src/amplify-exception-handler.ts

```diff
@@ -108,6 +108,23 @@ const printAmplifyException = (amplifyException: AmplifyException): void => {
   if (stack) {
     printer.debug(stack);
   }
+
+  if (isDebug && amplifyException.downstreamException) {
+    printer.blankLine();
+    if (amplifyException.downstreamException instanceof AmplifyException) {
+      printAmplifyException(amplifyException.downstreamException);
```

Comment:
I don't know that it matters much because the chain of downstreamExceptions is likely to be rather short, but I would prefer we just iterate down the list of downstreamException and delegate out to `printAmplifyException` or `printError` rather than recursively call `printAmplifyException`

Reply:
updated